docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,129 @@
+  timezone="Pacific/Honolulu"
+  label={'Rendered in "Pacific/Honolulu"'}
+/>
+```
+
+Please check the [full instructions](https://mui.com/x/react-date-pickers/timezone/) for more information on using this feature in your application.
+
+### Digital clock
+
+This feature is an alternative to the analog clock on time pickers; the original proposal was mainly focused on mobile devices, whereas the new interface is designed for a desktop experience or better time precision.
+This new addition makes our date pickers even more versatile and user-friendly and is available as the default variant for desktops on both [TimePicker](https://mui.com/x/react-date-pickers/time-picker/) and [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/).
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
+  <source src="/static/blog/mui-x-mid-v6-features/timepicker.mp4" type="video/mp4">
+</video>
+
+Check out all the possibilities in [documentation](https://mui.com/x/react-date-pickers/digital-clock/), and let us know your thoughts!
+
+## Data Grid
+
+### Copy and Paste[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+
+After setting the prop `experimentalFeatures={{ clipboardPaste: true }}` you can copy and paste data directly from and to your Data Grid.
+
+This new feature is extremely versatile, and you can exchange data from other fields, other grids, and even directly from and to Microsoft Excel.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/mui-x-mid-v6-features/copy_paste.mov" type="video/mp4">
+</video>
+
+This feature is integrated with the editing API, so pasted data can be persisted using the `processRowUpdate` prop to update your data source through your usual editing validation process.
+Furthermore, the callbacks `clipboardPasteStart` and `clipboardPasteEnd` are fired during the clipboard paste operation, which can be useful for extra customizations around the event.
+
+For more details on how to use clipboard copy and paste, check out the [feature's page](https://mui.com/x/react-data-grid/clipboard/).
+
+### Filter on column headers[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+This feature allows users to quickly filter the data without needing any additional menu and can be activated using the `unstable_headerFilters` prop.
+
+The new filter fields are displayed below the headers and are synchronized with the filter panel, but for those seeking more simplicity, disable the default filter panel using `disableColumnFilter` prop and set filters to use only the default operator.
+You can also customize the header filter cell for a specific column using the `renderHeaderFilter` method of `GridColDef`
+
+For more details on how to use and customize the filters, check out its [documentation page](https://mui.com/x/react-data-grid/filtering/header-filters/).
```

Comment:
This seems like a pretty major and awaited change, maybe it would make sense to showcase it visually as well? 🤔

Reply:
Added a video!